pkg/services/ngalert/image/service.go

```diff
@@ -24,16 +24,6 @@ const (
 	screenshotCacheTTL = 60 * time.Second
 )
 
-var (
-	// ErrNoDashboard is returned when the alert rule does not have a Dashboard UID
-	// in its annotations or the dashboard does not exist.
-	ErrNoDashboard = errors.New("no dashboard")
```

Comment:
Like Josh mentioned I thought we were moving just the interface? I'd like to keep the errors local to the `image` package, it feels weird that the image package returns `model` errors.

Reply:
I think this is a bit orthogonal to what I've stated - I don't have an opinion on where errors go, they look fine from here.

Reply:
I put them in this file because that's where `AlertRule` is defined. These errors are logically associated with alert rules. If a rule specifies an invalid dashboard UID, that is a problem with the rule, not necessarily something specific to images.

Specifically, we might want to use these errors in the dashboard annotations package. It seems similarly weird to have annotations depend on images simply because that's where the relevant errors are defined.

Reply:
Also, check the comment I left above - there are already errors outside of `ngalert` for these same issues (invalid UID, invalid panel ID), we could also just straight up delete them if we feel we want to.